.github/workflows/main.yml

```diff
@@ -32,8 +32,8 @@ jobs:
       NESSIE_DIR: included-builds/nessie
       NESSIE_MAIN_REPOSITORY: projectnessie/nessie
       NESSIE_MAIN_BRANCH: main
-      NESSIE_PATCH_REPOSITORY: ''
-      NESSIE_PATCH_BRANCH: ''
+      NESSIE_PATCH_REPOSITORY: snazy/nessie
+      NESSIE_PATCH_BRANCH: remove-superfluous-system-property-hack
```

Comment:
do we want to merge this and remove it right after?

Reply:
I'll remove it before merge